win32/Makefile.msc

```diff
@@ -30,9 +30,12 @@ WFLAGS  = \
 	-DX86_FEATURES \
 	-DX86_PCLMULQDQ_CRC \
 	-DX86_SSE2 \
+    -DX86_SSE42_ADLER32 \
```

Comment:
Identation here is messed up in these changes in this file. These makefiles may use tabs instead of spaces.